ctapipe/image/hillas.py

```diff
@@ -153,7 +153,7 @@ def hillas_parameters(geom, image):
         psi = skewness_long = kurtosis_long = np.nan
     else:
         if vx != 0:
-            psi = np.arctan(vy / vx)
+            psi = np.arctan2(vy, vx)
```

Comment:
This a a major change!

The usage of arctan was intentional, since you cannot have a preference here!
psi simply does not contain a directional information, which is the point of the head/tail disambiguation in disp regression.

This normalizes psi to the [-pi, pi] interval, to make this apperent.

Why was this changed?

Reply:
If this was intentional, it should be documented in a comment.   I guess the only place in the code that relies on the psi angle having a particular range would be in plotting, and there is should be modded to the correct value, right?  Defining hillas planes should have no effect on which quadrant the angle is defined in.  

Nevertheless, it's ok to change  this back, but I would strongly suggest adding a comment why, and a unit test if it is required to be in [-pi/2,pi/2] here. Otherwise, it could be added to the docstring of the psi Field, explaining the expected range.

Reply:
I think there was no test since the different precursor algorithms worked differently. But know we should add one, indeed.

Reply:
As far as I saw, it is nowhere required for psi to be in a particular range, only that is it to be measured counter-clockwise from the positive x-axis in the camera frame.

For the unit-test wouldn't then the one coming with this PR be sufficient? in the end psi is just used to reconstruct the core position.

Reply:
Probably it is better to revert this change, then?

Reply:
Yes, it's my fault - let's revert it just to avoid future confusion, but please also add a comment that it should give a psi in the range (-pi/2,pi/2).  It won't have an effect on reconstruction, but having it defined in a consistent way with no degenerate values makes it a bit easier to later apply a sign based on the asymmetry. 

This also reminds me of this PR: https://github.com/cta-observatory/ctapipe/issues/1078
But perhaps it's too late to rename it!

Reply:
is this PR closed? I don't see any PR open from that issue